pubspec.lock

```diff
@@ -333,7 +345,7 @@ packages:
       name: js
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.6.5"
```

Comment:
🤔did you mean to downgrade this package to make go_router work?

Reply:
ahm, not really, not sure why this happened, I'll investigate.

Reply:
I removed the `.lock` and re-generated it by `flutter pub get` and it indeed changed these dependencies including the downgrade.